packages/thirdweb/src/transaction/prepare-transaction.ts

```diff
@@ -17,6 +17,7 @@ export type StaticPrepareTransactionOptions = {
   maxFeePerGas?: bigint | undefined;
   maxPriorityFeePerGas?: bigint | undefined;
   maxFeePerBlobGas?: bigint | undefined;
+  feeType?: FeeType | undefined;
```

Comment:
actually lets not add this, lets just do the chain one.

this prop is too similar the tx.type one which also indicates if its a legacy or 1559 tx

Reply:
we still would want a way to tell at the transaction level, hey please calculate the gas and send as legacy. If not this interface, we can use a different one, any thoughts?

Reply:
we have `type` in `SerializableTransaction` already should map to that